docs/integrations/sms.md

```diff
@@ -4,8 +4,9 @@ sidebar_position: 3
 
 # SMS
 
-Infobip Adapter enables sending [WhatsApp](https://www.whatsapp.com/) and SMS messages by using [Infobip's APIs](https://www.infobip.com/docs/api/channels/whatsapp/whatsapp-outbound-messages) and integrating it with the Platform. 
-It can be used for facilitating communication with your users.
+Infobip Adapter enables sending and receiving SMS messages by using [Infobip's APIs](https://www.infobip.com/docs/api/channels/sms) and integrating it with the Platform. 
+It can be used for facilitating communication with your users. Keep in mind that in order to avoid blacklisting and filtering, [appropriate number type](https://www.infobip.com/docs/api/platform/numbers) should be used and [registered](https://www.infobip.com/docs/api/platform/numbers/number-registration).
```

Comment:
I would change the order in the second part of this sentence to something like this: "Keep in mind ..., use registered numbers and pay attention to the appropriate number type."